src/services/attendees/attendee-router.ts

```diff
@@ -18,6 +25,36 @@ attendeeRouter.post("/", async (req, res, next) => {
     }
 });
 
+// generates a unique QR code for each attendee
+attendeeRouter.get(
+    "/qr/",
+    RoleChecker([Role.Enum.USER]),
+    async (req, res, next) => {
+        const payload = res.locals.payload;
+
+        try {
+            const userId = payload.userId;
+            const expTime = Math.floor(Date.now() / 1000) + 20; // Current epoch time in seconds + 20 seconds
+            let hashStr = userId + "#" + expTime;
+            const hashIterations = Number(getEnv("QR_HASH_ITERATIONS"));
+            const hashSecret = getEnv("QR_HASH_SECRET");
```

Comment:
should be config.QR_HASH_SECRET and QR_HASH_SECRET should come from Config (which calls getEnv)